packages/react-native-bpk-component-navigation-bar/stories.js

```diff
@@ -35,6 +37,20 @@ import BpkNavigationBar, {
 
 const exampleLogo = require('./logo.png');
 
+/*
+HACK
+On iOS, Storybook's UI covers the status bar which affects the appearance
+of the navigation bar. This pulls it upwards to counteract it.
+*/
+const IOS_STATUS_BAR_HEIGHT = isIphoneX ? 44 : 20;
+const styles = StyleSheet.create({
+  navigationBar: Platform.select({
+    ios: {
+      marginTop: -IOS_STATUS_BAR_HEIGHT,
```

Comment:
Screenshots in this PR don't show iPhone X, so here's proof it works:

![simulator screen shot - iphone xs - 2018-12-20 at 09 43 34](https://user-images.githubusercontent.com/73652/50277239-c4a15600-043b-11e9-9d94-e50349025fc3.png)